i18n/es/docusaurus-plugin-content-docs/current/unraid-os/release-notes/7.2.0.md

```
@@ -42,7 +42,7 @@ Ahora puedes expandir tus pools RAIDZ1/2/3 de un solo vdev, una unidad a la vez.
 - Corrección: Ahora habrá una advertencia de "expansión inválida" si el pool necesita ser actualizado primero
 - Mejora: mejores ajustes predeterminados para los vdevs ZFS RAIDZ
 
-#### Soporte para Ext2/3/4, NTFS y exFAT {/* #ext234-ntfs-and-exfat-support */}
+#### Soporte para Ext2/3/4, NTFS y exFAT {/* #ext234-ntfs-and-exfat-support */} {/* #ext234-ntfs-and-exfat-support */}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)